### PR TITLE
beefed up gitignore and changed my personal page a little

### PR DESCRIPTION
**Pull request by albeanth**

### CODE REVIEW

File: users/albertia/index.md [hunk 1]
```diff
@@ -5,14 +5,19 @@ author: Tony Alberti
...
+
+Since then, I have been pursuing my Doctoral degree in further developing depletion methods for MAMMOTH - the reactor physics subpackage of the Multiphysics Object Oriented Simulation Environment, otherwise known as MOOSE.  In particular, I am focusing on reducing the order of the depletion calculations in order to observe an effective speedup in depletion calculations.
+
+I have interned at Idaho National Laboratory (INL) in Idaho Falls, ID in the summer of 2015 and 2016. At INL I worked with the Rattlesnake/MAMMOTH Reactor Physics and Analysis group, specifically under the guidance of Javier Ortensi.
 
 ### Research Interests
 1. Reactor analysis and methods
-2. Nuclear fuels analysis
-3. Reduced order modeling
+2. Reduced order modeling
```
Comment (dougwoods3) [inline users/albertia/index.md:19]: whoa, big changes.

**Comment by albeanth** [comment]
right?? I don't mess around, man.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,497 @@
-/_site
+#############################
```
Comment (dougwoods3) [inline .gitignore:1]: i assume these are the ``bundle install''

Comment (dougwoods3) [reply .gitignore:1]: just kidding. this is the .gitignore stuff

**Comment by dougwoods3** [comment]
:shipit:

**Pull request merged**